lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -1109,10 +1109,47 @@ export class StandardController implements IController {
     /**
      * Returns the configuration object
      */
-    getConfiguration(): BrowserConfiguration {
+    public getConfiguration(): BrowserConfiguration {
         return this.config;
     }
 
+    public getPerformanceClient(): IPerformanceClient {
+        return this.performanceClient;
+    }
+
+    public getBrowserStorage(): BrowserCacheManager  {
+        return this.browserStorage;
+    }
+
+    public getNativeInternalStorage(): BrowserCacheManager  {
```

Comment:
Why do we want to expose these?

Reply:
native internal storage is used by 1p lib.

Reply:
I meant, why `public` as `protected` works for 1P today since we derive it? Is this for future proofing where we want `1P` to be based on the `public interface` of 3P?

Reply:
We extend `PublicClientApplication`, not a controller. That's why I have made those methods public.

Reply:
Ah! Alright. Thanks.